specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/1.2-preview/iotcentral.json

```diff
@@ -0,0 +1,5643 @@
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/jobs/{jobId}": {
+      "get": {
+        "operationId": "Jobs_Get",
+        "summary": "Get a job by ID",
+        "description": "Get details about a running or completed job by job ID.",
+        "x-ms-examples": {
+          "Get a job by ID": {
+            "$ref": "./examples/jobs_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/JobId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Job"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "Jobs_Create",
+        "summary": "Execute a new job",
+        "description": "Create and execute a new job via its job definition.",
+        "x-ms-examples": {
+          "Execute a new job": {
+            "$ref": "./examples/jobs_create.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/JobId"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Job definition.",
+            "schema": {
+              "$ref": "#/definitions/Job"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Job"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/jobs/{jobId}/devices": {
+      "get": {
+        "operationId": "Jobs_GetDevices",
+        "summary": "Get device statuses",
+        "description": "Get the list of individual device statuses by job ID.",
+        "x-ms-examples": {
+          "Get device statuses": {
+            "$ref": "./examples/jobs_devices.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/JobId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/JobDeviceStatusCollection"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
```

Comment:
Operation might be pageable. Consider adding the x-ms-pageable extension.

Reply:
Good catch, missing this part, added.